code/modules/cm_marines/Marine_Research/BioPrinter.dm

```diff
@@ -99,7 +99,7 @@ Basically a cheap knock-off of the Protolathe that I wrote in the middle of the
 	icon_state = "protolathe"
 	busy = 1
 	use_power(max(1000, (3750)))
-	if (do_after(user, 16, TRUE, 5, BUSY_ICON_GENERIC))
```

Comment:
Is the removal of the icon intended here?

Reply:
same reason as protolathe, circuit printers.